contrib/local-environment/oauth2-proxy-alpha-config.cfg

```diff
@@ -1,10 +1,6 @@
 http_address="0.0.0.0:4180"
 cookie_secret="OQINaROshtE9TcZkNAm-5Zs2Pv3xaWytBmc5W7sPX7w="
-provider="oidc"
 email_domains="example.com"
-oidc_issuer_url="http://dex.localhost:4190/dex"
-client_secret="b2F1dGgyLXByb3h5LWNsaWVudC1zZWNyZXQK"
-client_id="oauth2-proxy"
 cookie_secure="false"
 
-redirect_url="http://localhost:4180/oauth2/callback"
+redirect_url="http://localhost:4180/oauth2/callback"
```

Comment:
Lost a new line here, can we resolve that?

Reply:
Still missing the trailing new line, perhaps your IDE is doing something weird here?

Reply:
pushing it again, if it doesn't catch this time could you please suggest the change and I will commit it?